<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidgetImageSelector</class>
 <widget class="QDockWidget" name="DockWidgetImageSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geovis Imagery Selector</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="QToolButton" name="btnAddToMap">
      <property name="toolTip">
       <string>Add selected imagery to the map.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="btnClear">
      <property name="toolTip">
       <string>Clear the rectangle and imagery list.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="btnSelArea">
      <property name="toolTip">
       <string>Select an area by clicking and dragging on the map.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
